galaxy/api/serializers/serializers.py

```diff
@@ -386,7 +386,11 @@ def get_summary_fields(self, obj):
         summary['task_messages'] = [OrderedDict([
             ('id', g.id),
             ('message_type', g.message_type),
-            ('message_text', g.message_text)
+            ('message_text', g.message_text),
+            ('content_id', g.content_id),
+            ('is_linter_rule_violation', g.is_linter_rule_violation),
+            ('linter_type', g.linter_type),
+            ('linter_rule_id', g.linter_rule_id)
```

Comment:
Can I get the short description for the rule here?

Reply:
Since we're working across 3 linters, I can parse the message text for everything but the filepath and rule id (for ansible-lint this ends up being the same as the short description in the rule class).

Reply:
I just need the message text. I don't want to display the filepath or line number on the detail page